front/lib/api/assistant/agent.ts

```diff
@@ -595,7 +595,7 @@ async function* runMultiActionsAgent(
         messageId: agentMessage.sId,
         error: {
           code: "multi_actions_error",
-          message: `Error running assistant: ${event.content.message}`,
+          message: `Error running assistant: [${event.content.code}] ${event.content.message}`,
```

Comment:
mmmmmmmm not sure this will give us the 429 vs 529 will it?

Reply:
oh i misread event as error

Reply:
nvm then, not sure we can do much if the code is not in the message

Reply:
Yep is has to be done in core 👍